build-data/generated/tracking_parameters.json

```diff
@@ -0,0 +1,902 @@
+                "content.time.com/time/world/article/0,8599,2054433,00.html",
+                "developer.amazon.com",
+                "www.engadget.com/yubico-yubikey-bio-fingerprint-reader-130034276.html",
+                "edmunds.com",
+                "psychcentral.com"
+            ],
+            "requests3p": {
+                "prevalence": 0.078,
+                "entities": {
+                    "Google LLC": 0.136,
+                    "id.a-mx.com": 0.075,
+                    "Sailthru, Inc": 0.065,
+                    "Adobe Inc.": 0.047,
+                    "ANIVIEW LTD": 0.04,
+                    "Bazaarvoice, Inc.": 0.04,
+                    "Akamai Technologies": 0.029,
+                    "Index Exchange, Inc.": 0.028,
+                    "Salesforce.com, Inc.": 0.024
+                }
+            },
+            "cookies": {
+                "prevalence": 0.001,
+                "firstParty": 25,
+                "thirdParty": 29,
+                "entities": {
+                    "RELX Group": 0.241,
+                    "concealedcarry.com": 0.138,
+                    "Paysafe Group Limited": 0.103,
+                    "The Nasdaq Stock Market LLC": 0.069,
+                    "Salesforce.com, Inc.": 0.069,
+                    "Auth0, Inc.": 0.069,
+                    "urban-vpn.com": 0.069,
+                    "expresso.pt": 0.069,
+                    "lawofselfdefense.com": 0.069
+                }
+            }
+        },
+        "hmb_campaign": {
+            "prevalence": 0.069,
+            "exampleSites": [
+                "www.nbcnews.com",
+                "ph.news.yahoo.com/yahoo-sportsbook-daily-ravens-at-dolphins-betting-preview-081036579.html",
+                "www.thesaurus.com/browse/way",
+                "www.usatoday.com/videos/travel/2021/08/25/construction-theme-park-dig-world-coming-katy-texas/5584901001",
+                "www.testberichte.de/test/produkt_tests_triangle_ventis_xs_p14548.html",
+                "vk.com/webgov",
+                "deals.usnews.com/coupons/windscribe",
+                "www.macworld.com",
+                "patch.com",
+                "www.thespruce.com"
+            ],
+            "requests3p": {
+                "prevalence": 0.069,
+                "entities": {
+                    "Google LLC": 0.141,
+                    "id.a-mx.com": 0.084,
+                    "Sailthru, Inc": 0.073,
+                    "ANIVIEW LTD": 0.038,
+                    "Akamai Technologies": 0.033,
+                    "Index Exchange, Inc.": 0.033,
+                    "Salesforce.com, Inc.": 0.028,
+                    "WarnerMedia, LLC": 0.017,
+                    "Amazon Technologies, Inc.": 0.015
+                }
+            },
+            "cookies": {
+                "prevalence": 0.001,
+                "firstParty": 33,
+                "thirdParty": 15,
+                "entities": {
+                    "concealedcarry.com": 0.267,
```

Comment:
the number here is no longer the same prevalence as above, right? This 0.267 is percentage of that 0.001?

Reply:
> This 0.267 is percentage of that 0.001?
correct